sdk/src/schema.ts

```diff
@@ -73,22 +79,21 @@ export type InvariantEventType =
 
 export interface TxResult {
   hash: string
-  events: InvariantEventType[]
 }
 
 export type CreatePositionTxResult = {
   hash: string
-  events: [CreatePositionEvent]
+  events: CreatePositionEvent[]
```

Comment:
Are you sure you want to incorporate array here? The `create_position` entrypoint always emits one event in case of success.

Just reminder of the difference:
- `events: [CreatePositionEvent]` - This syntax is using tuple type. In TS, tuple is array with **fixed number** of elements whose types are known, but need not be the same. In this case, the tuple is defined to have exactly one element of type `CreatePositionEvent`. It's quite specific in its length and the type of its elements
- `events: CreatePositionEvent[]` - This syntax defines an array of `CreatePositionEvent` type. Unlike a tuple, this array can have **any number of elements** (including zero), but all elements must be of the `CreatePositionEvent` type. It's more flexible in terms of the number of elements it can contain.